scripts/prepare-crds

```diff
@@ -65,18 +65,13 @@ name: ${name}-crd
 type: application
 EOF
 
-# Add providesGVR annotation
-providesGVR="$(cat ${f}/package.yaml | yq r - generateCRDChart.providesGVR)"
-if ! [[ -z ${providesGVR} ]]; then
-	if [[ -z "$(yq r ${f}/charts/Chart.yaml 'annotations[catalog.cattle.io/auto-install-gvr]')" ]]; then
-		yq w -i ${f}/charts/Chart.yaml "annotations[catalog.cattle.io/auto-install-gvr]" "${providesGVR}"
-	fi
-	if [[ -z "$(yq r ${f}/charts/Chart.yaml 'annotations[catalog.cattle.io/provides-gvr]')" ]]; then
-		yq w -i ${f}/charts-crd/Chart.yaml "annotations[catalog.cattle.io/provides-gvr]" "${providesGVR}"
-	fi
-	if [[ -z "$(yq r ${f}/charts/Chart.yaml 'annotations[catalog.cattle.io/hidden]')" ]]; then
-		yq w -i ${f}/charts-crd/Chart.yaml "annotations[catalog.cattle.io/hidden]" "true"
-	fi
+# Make the primary chart auto-install the CRD chart
+if [[ -z "$(yq r ${f}/charts/Chart.yaml 'annotations[catalog.cattle.io/auto-install]')" ]]; then
```

Comment:
Since this one is changed to update the primary chart, should it also be moved to the scripts/prepare file before or after providesGVR check?

Reply:
auto-install would only exist if there is a sub-chart